lib/streamlit/watcher/file_watcher.py

```diff
@@ -50,6 +39,24 @@
 ]
 
 
+def report_watchdog_availability():
```

Comment:
This wasn't previously tested, but now that it has been factored out into a helper function, it'd be great if we could add a small unit test for this in `file_watcher_test.py`

Reply:
updated